src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -287,6 +287,64 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyRepoArtifacts"
+          AfterTargets="Package"
+          Condition="Exists($(RepoManifestFile))">
+
+    <XmlPeek XmlInputPath="$(RepoManifestFile)"
+             Query="Build/Artifact/@Path">
+        <Output TaskParameter="Result" ItemName="ArtifactPath" />
```

Comment:
It's likely that this item will conflict with something defined in the Arcade SDK or the .NET SDK. Consider renaming to something more specific liike `RepoManifestArtifact`.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18153/commits/9bfacadeccfd5235e7e99c2d624cb92b7807b176